bitnami/orangehrm/README.md

```diff
@@ -191,6 +195,79 @@ Find more information about how to deal with common errors related to Bitnami’
 
 ## Upgrading
 
+### To 8.0.0
+
+In this major there were two main changes introduced:
+
+1. Adaptation to Helm v2 EOL
+2. Updated MariaDB dependency version
+
+Please read the update notes carefully.
+
+**1. Adaptation to Helm v2 EOL**
```

Comment:
The "1. ..." is at the same level as the entries inside, maybe we could use h4/h5 instead so it is easier to understand?

Reply:
I tried it at first but h4 is actually smaller than using bolds and h5 is smaller than the standard font. We also shouldn't use h3 as that would make too big. As a compromise, I used numbers to highlight these lines, though I actually prefer using bullet points.

Reply:
It just looks weird having multiple bolds with the same size, but up to you